File: Beecrowd/1047 Tempo de Jogo com Minutos/main.c
#include <stdio.h>
 
int main() {
 
	int hora_inicial, minuto_inicial, hora_final, minuto_final, hora_total, minuto_total;
	
	scanf("%d %d %d %d", &hora_inicial, &minuto_inicial, &hora_final, &minuto_final);
	
	hora_total = hora_final - hora_inicial;
	
	if (hora_total < 0)
    	hora_total += 24;
    
    minuto_total = minuto_final - minuto_inicial;
    
    if (minuto_total < 0) {
    	minuto_total += 60;
        if(hora_total != 0)
            hora_total -= 1;
        else
            hora_total = 23;
	}

	if (minuto_total == 0 && hora_total == 0)
    	printf("O JOGO DUROU 24 HORA(S) E 0 MINUTO(S)\n");
    else
    	printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", hora_total, minuto_total);
 
    return 0;
}

File: Beecrowd/1047 Tempo de Jogo com Minutos/main.py
entrada = [int(i) for i in input().split()]

hora_inicial = entrada[0]
minuto_inicial = entrada[1]
hora_final = entrada[2]
minuto_final = entrada[3]

hora_total = hora_final - hora_inicial

if hora_total < 0:
    hora_total += 24

minuto_total = minuto_final - minuto_inicial

if minuto_total < 0:
    minuto_total += 60
    if hora_total != 0:
        hora_total -= 1
    else:
        hora_total = 23

if minuto_total == 0 and hora_total == 0:
    print("O JOGO DUROU 24 HORA(S) E 0 MINUTO(S)")
else:
    print("O JOGO DUROU {} HORA(S) E {} MINUTO(S)".format(hora_total, minuto_total))